src/openrct2/ride/Vehicle.cpp

```diff
@@ -8969,22 +8964,7 @@ void Vehicle::UpdateTrackMotionMiniGolfVehicle(
     trackPos = { TrackLocation.x + moveInfo->x, TrackLocation.y + moveInfo->y,
                  TrackLocation.z + moveInfo->z + RideData5[curRide->type].z_offset };
 
-    // Investigate redundant code
```

Comment:
History note: it is proper code in the regular vehicle update track motion function - it was probably just a copy-paste. (Possibly even by CS himself.)